hnn_qt5.py

```diff
@@ -208,7 +220,8 @@ def runsim (self):
         simdat.ddat['dpltrials'] = readdpltrials(os.path.join(dconf['datdir'],paramf.split(os.path.sep)[-1].split('.param')[0]),self.ntrial)
         if debug: print("Read simulation outputs:",simdat.dfile.values())
 
-        simdat.updatelsimdat(paramf,simdat.ddat['dpl']) # update lsimdat and its current sim index
+        if not is_opt:
+          simdat.updatelsimdat(paramf,simdat.ddat['dpl']) # update lsimdat and its current sim index
```

Comment:
Trials during optimization mode are almost certainly meaningless. If we could save the trials from only the best run, that could be useful. Perhaps TODO in a future version.

Reply:
Isn't there a trend towards decreasing error over trials? Might be useful to display later on...

Reply:
Ah, true. However, this is dependent on the algorithm and it's hard to discern when the algorithm is making changes for exploration (increase RMSE) vs. refinement (decrease RMSE). Perhaps something to keep in mind for the future.